README.md

```diff
@@ -1,70 +1,49 @@
-# Getting Started with Create React App
+# Ratenect
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![Ratenect Banner](images/banner.png)
+<br><br>
+Ratenect is a web platfrom with a two-way rating system where the users can find NGOs near them and can apply to volunteer in-person for their public activities. Based on their experience the NGOs can rate the volunteers and vice-versa.
+<br><br>
+Read more details about the project inspiration [here](https://cbrtl.github.io/posts/ratenect/).
 
-## Available Scripts
+## Tech Stack
 
-In the project directory, you can run:
+- **Frontend**: React
+- **Backend**: Node js, Express js
+- **Database**: MongoDB (Atlas)
 
-### `npm start`
+## Plan
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+- **UI**: [Figma](https://www.figma.com/file/W7mgBV73gMNZRFjuEewyDN/ratenect?node-id=0%3A1)
+- **Database Schema**: [dbdiagram](https://dbdiagram.io/d/60e010ba0b1d8a6d39650956)
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## Run Project
 
-### `npm test`
+- Clone the repository
+- `cd` into the `ratenect-backend` folder
+- switch to dev branch `git checkout dev`
+- run `npm install`
+- run `npm install jsonwebtoken --save`
```

Comment:
This one is not needed to run the front-end, I guess. @eswarupkumar Please verify.

Reply:
i have edited this part acccording to frontend.. Done

Reply:
Ok.